webapp/channels/src/packages/mattermost-redux/src/utils/file_utils.ts

```diff
@@ -27,6 +27,10 @@ export function getFormattedFileSize(file: FileInfo): string {
     return `${bytes} B`;
 }
 
+export function formatBytes(bytes: number): string {
+    return getFormattedFileSize({size: bytes} as FileInfo);
```

Comment:
this seems wrong... 😂 
It would be nice if `getFormattedFileSize(file: FileInfo)` was actually `getFormattedSize(size: number)`

Reply:
I didn't want to change the existing functions, but if you think it's okay, then I can do that.

Reply:
Yah, I think it's worth it. Leave it nicer than when we started. :)